stanford/roles/cluster/defaults/main.yml

```diff
@@ -76,6 +77,11 @@ CLUSTER_SUBNETS:
       cidr: "{{ DEPLOYMENT_CIDR_BASE }}.45.0/24"
     - zone: 'us-west-1c'
       cidr: "{{ DEPLOYMENT_CIDR_BASE }}.46.0/24"
+  sandbox:
+    - zone: 'us-west-1a'
+      cidr: "{{ DEPLOYMENT_CIDR_BASE }}.185.0/24"
+    - zone: 'us-west-1a'
+      cidr: "{{ DEPLOYMENT_CIDR_BASE }}.186.0/24"
```

Comment:
We didn't conflict with this in one of the Insights PRs, did we?

Reply:
I think it's ok.